src/lib/orion_websocket/parser.cpp

```diff
@@ -30,10 +30,19 @@
 #include "rapidjson/writer.h"
 
 #include "logMsg/logMsg.h"
+#include "rest/HttpHeaders.h"
 
-void ws_parser_parse(std::string &url, std::string &verb, std::string &payload, const char *msg)
+
+#include <map>
```

Comment:
Ah, and another one from the style guide ... :-)
Always include one directory in the inclusion:

    #include "orion_websocket/parser.h"

Also not my idea, but it makes a lot of sense ...